extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/config/AstroThingConfig.java

```diff
@@ -36,6 +36,10 @@ public void parseGeoLocation() {
         if (geoParts.length == 2) {
             latitude = toDouble(geoParts[0]);
             longitude = toDouble(geoParts[1]);
+        } else if (geoParts.length == 3) {
+            latitude = toDouble(geoParts[0]);
+            longitude = toDouble(geoParts[1]);
+            altitude = Integer.parseInt(geoParts[2]);
```

Comment:
Why do you assume that it is an int? PointType uses BigDecimal for altitude, so you should support this.